frontend/src/Student/queries/shows.graphql

```diff
@@ -0,0 +1,15 @@
+query Shows {
+  shows(orderBy: { sort: "entryStart", direction: DESC }) {
+    id
+    name
+    entryStart
+    entryEnd
+    judgingStart
+    judgingEnd
+    entryCap
+    entries {
+      id
```

Comment:
Will we need more than just the id?

Reply:
I think so? Would the invited status be associated with the entries?